FirebaseABTesting.podspec

```diff
@@ -33,14 +33,14 @@ Firebase Cloud Messaging and Firebase Remote Config in your app.
   s.requires_arc = base_dir + '*.m'
   s.public_header_files = base_dir + 'Public/*.h', base_dir + 'Protos/developers/mobile/abt/proto/*.h'
   s.private_header_files = base_dir + 'Protos/developers/mobile/abt/proto/*.h'
+  s.preserve_paths = 'Interop/Analytics/Public/*.h', 'FirebaseCore/Sources/Private/*.h'
   s.pod_target_xcconfig = {
     'GCC_C_LANGUAGE_STANDARD' => 'c99',
     'GCC_PREPROCESSOR_DEFINITIONS' =>
       'GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1 ' +
       'FIRABTesting_VERSION=' + String(s.version),
     'HEADER_SEARCH_PATHS' => '"${PODS_TARGET_SRCROOT}"'
   }
-  s.dependency 'FirebaseAnalyticsInterop', '~> 1.3'
```

Comment:
I wonder if removing explicit versioned interop dependencies can make possible installing incompatible versions in case when developers lock particular pod versions? E.g. if FirebaseABTesting is locked to a version that requires FirebaseAnalyticsInterop  1.3  but a new Analytics version compatible with FirebaseAnalyticsInterop 2.0 is installed we may not get an error.

Reply:
There is no longer the concept of an Interop version. However, any breaking changes should be caught by CI.

Reply:
I see. I can imagine a case when an API change will be difficult to be caught by CI, e.g. if a new error is thrown but the method signature hasn't changed. Probably it's not too big concern but something to remember when making changes to API.

Reply:
Agreed. It should be standard practice to review all clients when making a breaking API change.

Reply:
That situation only gets caught now if the person remembers to update the podspec's major version and can only happen at a Firebase major version change.  In the future, if an interop header does a break all dependencies will also need a major version bump along with the Firebase pod major version bump.